components/clp-package-utils/clp_package_utils/scripts/native/search.py

```diff
@@ -8,6 +8,7 @@
 import socket
 import sys
 
+import netifaces
 import msgpack
```

Comment:
Nit: Sort the imports in lexical order.
```suggestion
import msgpack
import netifaces
```